src/components/NewToDoForm.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class NewToDoForm extends React.Component {
+  state = {
+    toDoTitle: '',
+    selectedUserId: '',
+    selectError: false,
+    inputError: false,
+  }
+
+  addTitle = (event) => {
+    this.setState({
+      toDoTitle: event.target.value,
+      inputError: false,
+    });
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+
+    const { toDoTitle, selectedUserId } = this.state;
+
+    if (!toDoTitle && !selectedUserId) {
```

Comment:
instead of 3 different if, you can check each prop separatety
if (!toDoTitle) do this
if (!selectedUserId) do this